test/parallel/test-http2-options-max-headers-block-length.js

```diff
@@ -22,6 +22,8 @@ server.listen(0, common.mustCall(() => {
   const client = h2.connect(`http://localhost:${server.address().port}`,
                             options);
 
+  client.on('error', () => {});
```

Comment:
Assuming `ECONNRESET` is the only expected flake, should it be more like this?
```suggestion
  client.on('error', err => {
    assert.strictEqual(err.code, 'ECONNRESET');
  });
```

Reply:
I didn't do it because it depends on the platform. It could also be `EPIPE`, `ECONNABORTED`, `ECANCELED` and it doesn't really matter. What matters is that the `'freameError'` and `'error'` events are emitted on the `req` object.